README.gr-GR.md

```diff
@@ -0,0 +1,16 @@
+<img width="150" height="150" align="left" style="float: left; margin: 0 10px 0 0;" alt="MS-DOS logo" src="https://github.com/Microsoft/MS-DOS/blob/master/msdos-logo.png">   
+
+# Πηγαίος Κώδικας του MS-DOS v1.25 και v2.0
+Αυτό το ταμείο κώδικα περιέχει τον πρωτότυπο πηγαίο-κώδικα και τα απανθισμένα δυαδικά αρχεία για το MS-DOS v1.25 και το MS-DOS v2.0.
```

Comment:
Instead of "Αυτό το ταμείο" you can write "Αυτή η αποθήκη".
Instead of "απανθισμένα" you can write "μεταγλωτισμένα".
And in general the sentences are not making a lot of sense.

Reply:
Thank you for your review. The wording used is valid translation and orthologically correct. Feel free to submit a separate PR with possible corrections to the initial translation.